### DISCUSSION TITLE
[MinimalLib] Can I set or get properties in sdf file using minimalLib ?

### DISCUSSION THREAD

**Discussion by seongsangcho-hits** [rdkit/rdkit#5369] [Q&A] [answered]
Hi,  SDWrite class have a `setProps` func in rdkitPython. But minimalLib does not.
So How can I set or get Properties in sdf file using minimalLib ?

**Comment by hhk7734** [comment, marked as answer]
Hey, I pushed PR https://github.com/rdkit/rdkit/pull/5414
